web/src/engine/websites/WebAce_e2e.ts

```diff
@@ -0,0 +1,25 @@
+﻿import { TestFixture, type Config } from '../../../test/WebsitesFixture';
+
+const config: Config = {
+    plugin: {
+        id: 'webace',
+        title: 'webエース (web ace)'
+    },
+    container: {
+        url: 'https://web-ace.jp/youngaceup/contents/1000188/',
+        id: '/youngaceup/contents/1000188/',
+        title: 'ひぐらしのなく頃に 巡'
+    },
+    child: {
+        id: '/youngaceup/contents/1000188/episode/8095/',
+        title: '第１８話-３'
+    },
+    entry: {
+        index: 0,
+        size: 317_431,
+        type: 'image/jpeg'
+    }
+};
+
+const fixture = new TestFixture(config);
+describe(fixture.Name, () => fixture.AssertWebsite());
```

Comment:
My be simplified to `describe(fixture.Name, fixture.AssertWebsite);`

Reply:
hum, well

![image](https://github.com/manga-download/haruneko/assets/10671518/32d1a509-0c0a-4af4-b908-872f138745ed)

i did gather all changes from master tho

Reply:
Mh, not sure how `describe` works internally, maybe it requires a closure for some access to outer variables.
Guess should be left untouched then...